src/core_plugins/kibana/public/discover/components/field_chooser/field_chooser.html

```diff
@@ -54,16 +54,16 @@ <h3 class="sidebar-list-header-label" id="available_fields" tabindex="0">Availab
         ng-class="{ 'kuiButton--basic': !filter.active, 'kuiButton--primary': filter.active, 'hidden-xs': !showFields, 'hidden-sm': !showFields }"
         class="kuiButton kuiButton--small pull-right discover-field-filter-toggle"
         ng-click="showFilter = !showFilter"
-        aria-label="Show field settings"
-        aria-haspopup="true"
-        aria-expanded="{{showFilter}}"
+        aria-label="{{showFilter ? 'Hide' : 'Show'}} field settings"
+        aria-expanded="{{!!showFilter}}"
+        aria-controls="discover-field-filter"
       >
         <span aria-hidden="true" class="kuiIcon fa-gear"></span>
       </button>
     </h3>
   </div>
 
-  <div class="sidebar-item discover-field-details" ng-show="showFilter">
+  <div class="sidebar-item discover-field-details" ng-show="showFilter" id="discover-field-filter">
```

Comment:
Absolutely not nit-picking, but me not reading the style-guide carefully enough. Thanks for pointing this out. Fixed it. The file itself (and I assume several others) contain quite some more ids not using camel-case. But I think, that needs to be addressed in a different PR.